SalesforceCommerceCloud/pwa-kit
# Import latest translations

We had added/updated the English text, and so recently we've received the translations for it. This PR imports those translations.

NOTE: these translations are considered "preliminary". The translators would like to do a round of testing. Any tweaks/fixes to the translations will come afterwards, if necessary.

# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- imported the translations
- compiled them too

# How to Test-Drive This PR

Just for sanity check, you may double check that the text for "Added to Wishlist" is translated:

1. Load the retail-react-app
2. Sign in
3. Navigate to a PDP
4. Switch to a different language, other than English, via the footer's dropdown
5. Then click on a button that adds the product to your wishlist
6. There's a message that pops up in the upper-right corner of the page
7. Verify that it's in fact translated and not in the default English.

# Checklists



## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
